.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,76 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+<!--
+Further instructions on how to fill a bug refer here:
+https://martinos.org/mne/stable/faq.html#i-think-i-found-a-bug-what-do-i-do
+
+If your issue is a usage question, submit it here instead:
+- The mne-python gitter: https://gitter.im/mne-tools/mne-python
+- The mne-analysis mailing list: https://mail.nmr.mgh.harvard.edu/mailman/listinfo/mne_analysis
+-->
+
+#### Describe the bug
+<!--
+Please provide a clear and concise description of what the bug is.
+-->
+
+#### Steps/Code to Reproduce
+<!--
+Please provide a code snippet or Minimal Working Example (MWE) to replicate your
```

Comment:
Include the link with the text Minimal Working Example (MWE).

Reply:
its there. You mean right after `(MWE) http... to replicate your proble.` It really breaks the read.

Reply:
I mean inline [Minimal Working Example (MWE)](https://en.wikipedia.org/wiki/Minimal_Working_Example)

Reply:
Also, let's not capitalize minimal working example.